src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+public class Ui {
+    private static final String startMessage = "____________________________________________________________\n"
+            + "Hello! I'm Duke. \nFor "
+            + "events and deadlines, be sure to state the due date or the event timing as such: "
+            + "'XXX /by dd-MM-yyyy HH:mm'\n"
+            + "____________________________________________________________";
+    private static final String endMessage = "____________________________________________________________\n"
+            + "Bye! Hope to see you again soon!\n"
+            + "____________________________________________________________";
+
+    private static final String taskDoesNotExistError = "____________________________________________________________\n"
+            + "☹ OOPS!!! The task you chose does not exist. Use the 'list' command to check the items in your list.\n"
+            + "____________________________________________________________";
+
+    /**
+     * Sends a customised error message, usually from DukeExceptions.
+     * @param message the error message that is printed out to the user.
+     */
+    public static void customErrorMessage(String message) {
+        System.out.println(message);
+    }
+
+    /**
+     * Sends Duke's start message when Duke awakens.
+     */
+    public static void sendStartMessage() {
+        System.out.println(Ui.startMessage);
+    }
+
+    /**
+     * Sends Duke's end message when Duke goes to sleep.
+     */
+    public static String sendEndMessage() {
+        return Ui.endMessage;
+    }
+
+    /**
+     * Prints a message to the user about the task that the user just deleted.
+     * @param task The task that got deleted.
+     * @param taskListSize The updated TaskList size.
+     */
+    public static String deletedTaskMessage(TaskItem task, int taskListSize) {
+        String body = "";
+        if (taskListSize == 1) {
+            /* System.out.println("Now you have " + 1 + " task in the list."); */
+            body = "Now you have " + 1 + " task in the list.\n";
```

Comment:
It might be simpler to just type the entire thing as one string instead of concatenating multiple.